layouts/partials/tw/list-3-col-story-nobyline.html

```diff
@@ -0,0 +1,69 @@
+{{ $rows := .rows | default -1 }}
```

Comment:
The other way to do this is to see if we can pass "no byline" to the existing list-3-col-story.html. But let's work on merging this first and then do a refactor where we combine common templates afterwards because it's easy to end up with a soup of if-statements if you simplify things the wrong way.